components/logiVehicle/fn_canPlayerUseLogi.sqf

```diff
@@ -20,23 +20,25 @@ private _logiType = GET_LOGITYPE(_logiVic);
 
 if (_logiType isEqualTo "") exitWith {false};
 
-private _failedCheck = false;
+// Default to true for a more pleasurable code reading experience
+private _passedCheck = true;
```

Comment:
Inverted the logic to make it easier to understand. Defaults to true. If they have role access -- great, still true, we break out true. If not, `_passedCheck` turns `false` and we move on the the group access check. Finally, we return the value of that check.